python/ray/train/torch/config.py

```diff
@@ -14,6 +14,11 @@
 from ray.train.constants import DEFAULT_NCCL_SOCKET_IFNAME
 from ray.util import PublicAPI
 
+from ray._private.accelerators.hpu import HPU_PACKAGE_AVAILABLE
```

Comment:
Let's do ray train changes in a separate PR. Only ray core changes in this PR.

Reply:
I think, we need a mechanism to generalize this for accelerators.

Reply:
Yes, we are designing that for Train.

Reply:
https://github.com/ray-project/ray/pull/40695